Add support for ephemeral storage in Azure clusters (#333)
apache/fluo-muchos: docs/azure-ephemeral-disks.md

> Using ephemeral storage within clusters deployed by Muchos for Azure

Author: Sure thing I followed your precedent with the notes about using Ubuntu. This approach is cleaner and allows the reader of the main README to focus on the big picture.

--------------------------------------------------------------------

> By default for Azure based clusters, Muchos will create 3 data disks, each of size 128GiB, attached to each VM. This provides

Author: Done, I added a relevant link as well.

persistent storage which ensures that the data in HDFS is safe and consistent even if the VMs are deallocated (stopped).

> However, certain cases, in case you'd like to use only the ephemeral / temporary disk storage for HDFS, you first need to 

Reviewer: ```suggestion
However, if you'd like to use only the ephemeral / temporary disk storage for HDFS, you first need to 
```

Author: Thanks @keith-turner for catching the grammatical issue and proposing the edit.
